appveyor.yml

```diff
@@ -6,11 +6,10 @@ environment:
   MINICONDA: C:\Miniconda36-x64
```

Comment:
Except the name is still 3.6

Reply:
doesn't really matter - conda will install the correct python version bc. of the depencies - but i'll change to make it more explicit!